blockchain/Dockerfile

```diff
@@ -28,10 +28,20 @@ CMD ./nodeos.sh
 ### Intialize the blockchain
 FROM tonomy_blockchain_base as tonomy_blockchain_initialized
 
+## Run bootstrap script
 RUN bash -c "./nodeos.sh &" && sleep 10 && ./initialize-blockchain.sh && ./nodeos.sh stop
 
 ### Install nodejs
 FROM tonomy_blockchain_initialized as tonomy_blockchain_nodejs
 
 RUN wget nodejs.org/dist/v18.12.1/node-v18.12.1-linux-x64.tar.gz
 RUN tar -C /usr/local --strip-components 1 -xzf node-v18.12.1-linux-x64.tar.gz
+
+### Easy cleos script
+FROM tonomy_blockchain_initialized as tonomy_blockchain_easycleos
+
+RUN mkdir /var/easycleos
+
+# Copy the script to the container
+RUN cp ./create-key.sh /var/easycleos/create-key.sh
```

Comment:
better to use COPY instead of RUN

Reply:
as COPY only executes if file was changed

Reply:
this is not working it throws error i tried it works with RUN cp as we did for config.ini file